doc/source/whatsnew/v0.21.0.txt

```diff
@@ -473,6 +473,7 @@ Performance Improvements
 - Improved performance of :meth:`Categorical.set_categories` by not materializing the values (:issue:`17508`)
 - :attr:`Timestamp.microsecond` no longer re-computes on attribute access (:issue:`17331`)
 - Improved performance of the :class:`CategoricalIndex` for data that is already categorical dtype (:issue:`17513`)
+- Improved performance of :meth:`RangeIndex.min` and :meth:`RangeIndex.max` by using ``RangeIndex`` properties to perform the computations (:issue:`17607`)
```

Comment:
Those links won't work because the pages are not generated (only the class docstring is included in the api). 
To solve this, we would need to find a way to specify which of the methods to include of which not (or list them manually in the api.rst, that is probably easier)